src/main/java/org/traccar/protocol/Gl200TextProtocolDecoder.java

```diff
@@ -197,9 +205,14 @@ private Object decodeInf(Channel channel, SocketAddress remoteAddress, String se
                 break;
         }
 
+        position.set(Position.KEY_ICCID, parser.next());
         position.set(Position.KEY_RSSI, parser.nextInt());
 
-        parser.next(); // odometer or external power
+        if (protocolVersion.startsWith("6E")) { // GV310LAU
```

Comment:
Are you sure there are no conflicts with other devices?

Reply:
Based on the existing tests and the documentation there aren't. I think it's a better option than using an attribute.

This also confirms it:
https://github.com/jaayesta/queclink-parser/blob/master/src/utils.js

Reply:
I don't really want to have two different ways of device identification. Do you want to update other code to use protocol version as well?

Reply:
you mean for GV355CEU?

Reply:
There are several places where we use model name.

Reply:
yes I see. Ok, I'll update for all.

Reply:
Also, the plan is to slowly migrate this decoder away from regex patterns. If you need to make big changes, it might make sense to migrate as well.

Reply:
I think we can do it without big changes. 

I'll make a commit soon to see if I'm on the right direction.

Reply:
should I leave the device name and  attribute as a fallback when we can't identify the model?